mars/oscar/backends/ray/pool.py

```diff
@@ -302,3 +303,11 @@ async def start(self):
 
     def mark_service_ready(self):
         self._state = RayPoolState.SERVICE_READY
+
+
+if ray:
```

Comment:
This will bring errors when ray is installed but not used in user code. Try call this line in locations where users **must** call when using Mars on ray, for inst., `new_cluster_in_ray`.

Reply:
> #2768 seems another ray client use case, can it be fixed by this PR as well?

It's also fixed in this PR.